Documentation/configuration.md

```diff
@@ -1,70 +1,60 @@
 # rkt configuration
 
-`rkt` reads configuration from two directories - a **system directory** and a
-**local directory**. The system directory defaults to `/usr/lib/rkt` and the
-local directory `/etc/rkt`, but both can be overridden via command-line flags.
-
-`rkt` looks for configuration files inside subdirectories of these two
-directories. It ignores everything but regular files with `.json` extension.
-This means `rkt` does _not_ search for the files by recursively going down
-the directory tree. This also means that users are free to put some additional
-files there (e.g. documentation).
-
-Every configuration file has two common fields: `rktKind` and
-`rktVersion`. Both are strings, and the rest of the fields are specified by
-that pair. The currently supported kinds and versions are described below.
+`rkt` reads configuration from two directories - a **system directory** and a **local directory**.
+The system directory defaults to `/usr/lib/rkt` and the local directory `/etc/rkt`, but both can be overridden via command-line flags.
+
+`rkt` looks for configuration files inside subdirectories of these two directories.
+It ignores everything but regular files with `.json` extension.
+This means `rkt` does _not_ search for the files by recursively going down the directory tree.
+This also means that users are free to put some additional files there (e.g. documentation).
+
+Every configuration file has two common fields: `rktKind` and `rktVersion`.
+Both are strings, and the rest of the fields are specified by that pair.
```

Comment:
Both fields' values are strings, and the subsequent fields are specified by this pair.